main.cpp

```diff
@@ -275,296 +300,283 @@ struct Bakery
         bool burn(int bakingTime);
     };
 
-    //bake bread
-    void bakeBread(double flourAmount, RyeBread brd);
-    //sell cakes
+    void bakeBread(double bFlourAmount, RyeBread brd);
     float cakeSell(std::string cakeName);
-    //smell great
     void smellGreat();
 
     RyeBread breadToSell;
 };
 
-/*
-Thing 3) aqua park
-5 properties:
-    1) number of pools (int)
-    2) number of saunas (int)
-    3) total length of water slides (double)
-    4) opening hours (float)
-    5) ticket price (float)
-3 things it can do:
-    1) relax muscles
-    2) be fun
-    3) learning swimming
- */
+void Bakery::bakeBread(double bFlourAmount, RyeBread brd)
+{
+    std::cout << "Flour used: " << bFlourAmount << std::endl << "Timer set to: " << brd.bakingTime << std::endl;
+}
+
+float Bakery::cakeSell(std::string cakeName)
+{
+    if (cakeName == "Sweet Cake")
+    {
+        std::cout << "Product sold: " << cakeName << std::endl;
+        return 9430.f;
+    }
+    else
+        return 0.f;
+}
+
+void Bakery::smellGreat()
+{
+}
 
 struct AquaPark
 {
-    //number of pools (int)
     int numPools = 5;
-    //number of saunas (int)
     int numSaunas = 3;
-    //total length of water slides (double)
     float totalWaterSlidesLength = 121.56f;
-    //opening hours (float)
     float openAt = 9.30f;
-    //ticket price (float)
     float regularTicker = 13.21f;
 
-    //relax muscles
     void relaxMuscles(bool goToSauna, int swimmingIntensity);
-    //be fun
     void haveFun(float totalWaterSlidesLength, int numberOfSlides);
-    //learning swimming
     bool learnToSwim(int age, int timeSpent);
 };
 
-/*
-Thing 4) house
-5 properties:
-    1) number of windows (int)
-    2) size of living room (float)
-    3) room height (float)
-    4) number of bathrooms (int)
-    5) total floor size (float)
-3 things it can do:
-    1) provide shelter
-    2) provide rest
-    3) get dirty
- */
+void AquaPark::relaxMuscles(bool goToSauna, int swimmingIntensity)
+{
+    goToSauna = false;
+    if (swimmingIntensity > 5)
+        goToSauna = true;
+        std::cout << "You should use one of the sauna's to relax your muscles!" << std::endl;
+}
```

Comment:
This is not python.
This is C++.
Scope is determined by curly braces, not indentation.
the `std::cout` statement will execute regardless of the `swimmingIntensity` value
If you want the std::cout to only execute when `swimmingIntensity > 5`, you need to wrap the `if()` body in curly braces
```
if( conditional )
{
    all();
    the();
    stuff();
    to();
    execute();
}
```
- [ ] revise